test-files/initVaultDev.sh

```diff
@@ -3,11 +3,23 @@ export VAULT_ADDR=http://localhost:8200
 export VAULT_TOKEN=my-dev-root-vault-token
 export VAULT_VERSION=${1:-1.0.1}
 
+case "$(uname -s)" in
+  Darwin*)
+    os="darwin_amd64"
```

Comment:
Since we are intending to contribute these changes upstream, I would rename the var to `OS` to be more consistent with the rest of the script.